docs/application/web/guides/applications/service-app.md

```diff
@@ -1,239 +1,185 @@
-| [Tizen](../../api/latest/device_api/tv/tizen/tizen.html) | The base object for accessing Tizen TV Web Device APIs. |
-| [Alarm](../../api/latest/device_api/tv/tizen/alarm.html) | This API provides methods for setting and unsetting alarms. |
-| [Application](../../api/latest/device_api/tv/tizen/application.html) | This API provides information about the currently-running and installed applications and ways to launch other applications.<br>Note that the `getRequestedAppControl()` method is only valid inside the `onRequest()` callback. |
-| [Package](../../api/latest/device_api/tv/tizen/package.html) | This API provides methods to install and uninstall Tizen packages and to get information about installed packages. |
-| [Filesystem](../../api/latest/device_api/tv/tizen/filesystem.html) | This API provides methods to access the file system of a device and to read, write, copy, move, and delete files. |
-| [Message Port](../../api/latest/device_api/tv/tizen/messageport.html) | This API provides methods for an application to communicate with other applications. |
-| [System Information](../../api/latest/device_api/tv/tizen/systeminfo.html) | This API provides information about the device's display, network, storage, and other capabilities. |
-
-## Related Information
-* Dependencies
-  - Tizen 3.0 and Higher for TV
+  <!DOCTYPE html>
+  <html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <meta name="description" content="Tizen Web UI Web service Template" />
+
+    <title>Tizen Web UI Web service Template</title>
+
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <script src="js/main.js"></script>
+    <script type="module">
+    import * as service from 'wrt:service';
+
+    var pkg_id = tizen.application.getCurrentApplication().appInfo.packageId;
+    var service_id = pkg_id + ".Service";
+
+    function startService() {
+      service.startService(service_id).then(function() {
+        alert("Succeeded to start service");
+      }, function(error) {
+        alert("Failed to start service : " + error);
+      });
+    }
+    function stopService() {
+      service.stopService(service_id).then(function() {
+        alert("Succeeded to stop service");
+      }, function(error) {
+        alert("Failed to stop service : " + error);
+      });
+    }
+
+    document.querySelector('#start_service').addEventListener('click', startService);
+    document.querySelector('#stop_service').addEventListener('click', stopService);
+    </script>
+  </head>
+
+  <body>
+    <div id="main" class="page">
+      <div class="contents">
+        <span>
+          <button id="start_service" class="service_button">Start service</button>
+        </span>
+        <span>
+          <button id="stop_service" class="service_button">Stop service</button>
+        </span>
+      </div>
+    </div>
+  </body>
+
+  </html>
+  ```
```

Comment:
The related information is removed.
Please add it back.

```
## Related Information
- Dependencies	
  - Tizen 3.0 and Higher for TV
```

Reply:
Done